### PR TITLE
Update HTML  assignments

### PR DESCRIPTION
**Pull request by Kailas2023**

### CODE REVIEW

File: kailas/Aboutme.html [hunk 1]
```diff
@@ -1,9 +1,10 @@
 <!DOCTYPE html>
 <html lang="en-us">
+  <title>Document<title>
```
Comment (KomalShinde813) [inline kailas/Aboutme.html:3]: one tab space required

File: kailas/Aboutme.html [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
 <head>
+  <h1>Kailas A Neel</h1>
```
Comment (KomalShinde813) [inline kailas/Aboutme.html:5]: one tab space required

File: kailas/Aboutme.html [hunk 3]
```diff
@@ -83,11 +84,17 @@
 <div class="kk">
 <center>
       <h1>Kailas A Neel</h1>
-      <h3>kailasneel007@gmail.com</h3>
+  <p>kailasneel007@gmail.com</p>
```
Comment (KomalShinde813) [inline kailas/Aboutme.html:87]: one tab space required

File: kailas/Aboutme.html [hunk 4]
```diff
@@ -83,11 +84,17 @@
...
 </center>
 </div>
 <div>
 <h2>About Me</h2> 
+                 <p><h3>I am Kailas Ankush Neel,I am Pursuing MCA under the department of
```
Comment (KomalShinde813) [inline kailas/Aboutme.html:92]: no need of this space

File: kailas/Aboutme.html [hunk 5]
```diff
@@ -97,6 +104,14 @@ <h2>About Me</h2>
  
 <div>
     <h2>Work</h2>
+
+                 <p><h3>I have done my two project in acdemic salesforce and advance java Language.</p>
```
Comment (KomalShinde813) [inline kailas/Aboutme.html:108]: remove unwanted space

File: kailas/Li-Attribute.html [hunk 6]
```diff
@@ -1,30 +1,32 @@
...
-        <dt>MEAN.
-               <dd>Mongo Express Angular Nodejs.</dd> </dt>
-       <dt>java
-                <dd>java is a programming language.</dd></dt>          
-</dl>
-</body>
-</html>
+<!DOCTYPE html>
+<html>
+  <title>Document</title>
```
Comment (KomalShinde813) [inline kailas/Li-Attribute.html:3]: one tab space required

File: kailas/Li-Attribute.html [hunk 7]
```diff
@@ -1,30 +1,32 @@
...
+<head>
+<h1>li attribute</h1>
+  </head>
+<body>
+    <ol>
+          <li>Mumbai</li>
```
Comment (KomalShinde813) [inline kailas/Li-Attribute.html:9]: one tab space required changed it in whole file

File: kailas/webpage1.html [hunk 8]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+<h1>li attribute</h1>
+<body>
+    <ol>
+          <li>Mumbai</li>
```
Comment (KomalShinde813) [inline kailas/webpage1.html:7]: read all coading rules make changes according to it

File: kailas/webpage1.html [hunk 9]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
```
Comment (KomalShinde813) [inline kailas/webpage1.html:3]: head is child of HTML so that need one space from <html>

File: kailas/webpage1.html [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+
+<dl>
+        <dt>MEAN
+               <dd>Mongo Express Angular Nodejs</dd> </dt>
+       <dt>java
+                <dd>java is a programming language</dd></dt>          
+</dl>
+</body>
+</head>
+</html>
```
Comment (KomalShinde813) [inline kailas/webpage1.html:31]: new line at the end of file required

**Comment by Kailas2023** [comment]
yes mam

**Comment by Kailas2023** [comment]
All files updated

**Pull request merged**